Makefile

```diff
@@ -0,0 +1,14 @@
+run:
+	docker run -d -p 8080\:8080 --restart always --name token-dispenser -v $(shell pwd)\:/var/tmp/token-dispenser/ token-dispenser
```

Comment:
`-v $PWD:/var/tmp/token-dispenser/` works great

Reply:
on debian 8.9 it doesn't work, changed on ${CURDIR};
https://stackoverflow.com/questions/3050231/how-can-i-capture-the-current-directory-as-an-absolute-pathname-in-a-make-variab